src/main/java/com/powsybl/openloadflow/network/impl/LfBusImpl.java

```diff
@@ -87,4 +87,9 @@ public void updateState(boolean reactiveLimits, boolean writeSlackBus, boolean d
     public boolean isParticipating() {
         return participating;
     }
+
+    public void setIsParticipating(boolean b) {
```

Comment:
Change for `setParticipating` and if you change that attribute, do not forget to store it and to restore it after contingency.